app/src/main/java/ghcharactertracker/com/ghcharactertracker/Character.java

```diff
@@ -0,0 +1,175 @@
+package ghcharactertracker.com.ghcharactertracker;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by jesse.mailhot on 3/6/2018.
+ */
+
+public class Character implements Parcelable{
+    String playerName = "";
+    int level, maxHealth, curExp, nextLevelExp, money;
+    long id;
+    CharClass charClass;
+    Scenario currentScenario;
+
+    public Character(CharClass charClass) {
+        this.playerName = "";
+        this.charClass = charClass;
+        this.level = 0;
```

Comment:
If you wanted to you could here and in the next constructor remove setting the ints to 0 and the currentScenario to null since those are the default values anyways.

If you want to set them maybe you could have a common initialize method that both constructors call.